projects/backend/controllers/render.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from 'express'
+import fetch from 'node-fetch'
+
+const fetchRenderedArticle = async (url: string) => {
+    const response = await fetch(url, {
+        headers: {
+            Accept:
+                'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
+        },
+    })
+    const responseBody = await response.text()
+    return {
+        success: response.statusText === 'OK',
+        status: response.status,
+        body: responseBody,
+    }
+}
+
+// TODO: this needs a test once we're happy with the correct format for the paths
+const replaceImageUrls = (html: string): string => {
+    return html.replace(/https:\/\/i.guim.co.uk\/img\//g, `../media/`)
+}
+
+export const renderController = async (req: Request, res: Response) => {
+    const path = req.params.path
+    const renderingUrl = `${process.env.APPS_RENDERING_URL}${path}?editions`
```

Comment:
Do we want to merge this with this `path` which doesn't work for all the articles?

Reply:
good point. If we merged it I could merge the archiver PR and start uploading HTML files to the S3 bucket which might help unblock the client side work even if it doesn't work for all articles

Reply:
(but I'm happy to wait until we have a backend that works for everything)